packages/node_modules/@ciscospark/internal-plugin-encryption/src/encryption.js

```diff
@@ -34,13 +34,13 @@ const Encryption = SparkPlugin.extend({
       });
   },
 
-  decryptScr(key, cipherScr) {
-    return this.getKey(key)
+  decryptScr(key, cipherScr, onBehalfOf) {
```

Comment:
What is `onBehalfOf`? Please document the type of parameter it is and an example.

Reply:
Documentation added

Reply:
@mattnorris gentle reminder